public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -523,6 +539,26 @@ public function metabox_save( $post_id, $post, $verify_nonce = true ) {
 					update_post_meta( $post_id, $key, $new_value );
 					break;
 
+				case 'select-streaming':
+					$allowed_values = array_keys( self::get_streaming_services() );
+					$key_other = wcpt_key_to_str( $key, 'wcpt_' ) . '-other';
+					if ( in_array( $values[ $key ], $allowed_values ) ) {
+						update_post_meta( $post_id, $key, $values[ $key ] );
+
+						if ( ! empty( $_POST[ $key_other ] ) ) {
+							update_post_meta(
+								$post_id,
+								$key_other,
+								sanitize_text_field( $_POST[ $key_other ] )
+							);
+						}
+					} else {
+						// The value isn't in the allowed values (anymore?) so we should save it as "other".
+						update_post_meta( $post_id, $key, 'other' );
+						update_post_meta( $post_id, $key_other, $values[ $key ] );
```

Comment:
the values in `$values` are [already escaped by `esc_attr`,](https://github.com/WordPress/wordcamp.org/blob/bad61fae01bf264bad736b24ebbfcadd3de95c5e/public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php#L490) so we don't need to do this twice. I only had to do it above because I was using `$_POST` directly.

Reply:
👍